tests/test_claude_wrapper_hooks.py

```
@@ -458,6 +458,44 @@ def test_plain_claude_launch_argv_has_no_empty_argument(failures: list[str]) ->
+        searched_dir = searched_dir.resolve()
+
+        wrapper = wrapper_dir / "claude"
+        shutil.copy2(SOURCE_WRAPPER, wrapper)
+        wrapper.chmod(0o755)
+
+        env = os.environ.copy()
+        env["PATH"] = f"{wrapper_dir}:{searched_dir}:/usr/bin:/bin"
+        env.pop("CMUX_CUSTOM_CLAUDE_PATH", None)
+
+        proc = subprocess.run(
+            ["claude", "--version"],
```

Ruff (0.15.12)

[error] Starting a process with a partial executable path

(S607)